config/config.go

```diff
@@ -0,0 +1,109 @@
+package config
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/containers/image/v5/types"
+	"github.com/joho/godotenv"
+)
+
+type ServerConfig struct {
```

Comment:
remove this struct, and instead use two vars for the data.
In addition, make them private and create getters that returns only by value